rskj-core/src/test/java/co/rsk/peg/federation/NonStandardErpFederationsTest.java

```diff
@@ -465,10 +459,10 @@ void createdRedeemScriptProgramFromNonStandardErpBuilder_withRealValues_equalsRe
     @Test
     void createdFederationInfo_withRealValues_equalsExistingFederationInfo_testnet() {
         // values from last real non-standard erp fed in testnet
-        BridgeConstants bridgeTestNetConstants = BridgeTestNetConstants.getInstance();
-        networkParameters = bridgeTestNetConstants.getBtcParams();
-        emergencyKeys = bridgeTestNetConstants.getErpFedPubKeysList();
-        activationDelayValue = bridgeTestNetConstants.getErpFedActivationDelay();
+        FederationConstants federationTestNetConstants = FederationTestNetConstants.getInstance();
```

Comment:
and have `federationTestnetConstants` at a class level?

Reply:
I tried doing something similar but everything is a bit tangled, wouldn't get into those changes right now